components/service/firefox-accounts/README.md

```diff
@@ -221,7 +221,7 @@ Once the configuration is available and an account instance was created, the aut
 
 ```kotlin
 launch {
-    val url = account.beginOAuthFlow(scopes, wantsKeys).await()
+    val url = account.beginOAuthFlow(scopes).await()
```

Comment:
This reminds me that I need to audit the APIs in components I've worked on to see if those READMEs have changed. 😅